CMakeLists.txt

```diff
@@ -187,6 +187,7 @@ if (USE_DEBUG_MALLOC AND WIN32)
 endif ()
 
 option(ADD_WINDOWS_ENCLAVE_TESTS "Build Windows enclave tests" OFF)
+# Warning: turning on simulation mode on Windows may cause test failures and random crashes
 option(WIN32_SIMULATION "Windows Simulation Mode" OFF)
```

Comment:
This is fine.

Anyways, my request was to make a note about simulation mode being currently disabled in the GettingStarted.Windows.md guide and also reference the issue number. Can you make that change please?

Reply:
Sure.